docs/building-blocks/index.rst

```diff
@@ -1,8 +1,8 @@
 .. _blocks-index:
 
-========================
-Standard Building Blocks
-========================
+==================================
+Common workflows and helpful tools
+==================================
```

Comment:
The title is updated, but not the body or folder name, which makes the title seem a bit off.

Maybe this change can be left out or followed through a bit for consistecy? Happy to see adjustment of the terminology of "standard building blocks", but I think its relevant to convey that we want to define a few ways of doing things in our jupyterhub org github repos centrally here. Currently with this suggestion, I think the `helpful` wording is a bit off. I figure `helpful tools` is like a neutral advice, while `recommended` is a clear suggestion, and `standard` is a prescription of how to do something. I see this section as at least a recommendation, possibly as a standard currently, and that is why the `helpful` adjective seems a bit off.

I figure that this text could be integrated under docs/team/repository-code-standards.md in some way in the future, or perhaps be de-nested from docs//team and become docs/repository-code-standards.

Reply:
This makes me feel like we should cleanly separate *team policy / standards* from *team recommendations / guides*. Let me try re-organizing the docs in that fashion and see how that looks. I think that'll mean re-moving the "Team contributor guide" to its own section